src/test/java/nl/knaw/dans/validatedansbag/core/service/OriginalFilepathsServiceImplTest.java

```diff
@@ -86,4 +91,12 @@ void getMapping_should_return_empty_result_if_original_filepaths_txt_does_not_ex
 
         assertEquals(0, result.size());
     }
+
+    @Test
+     void getSecurePath_should_throws_IllegalArgumentException(){
+        Path basePath = Path.of(OriginalFilepathsServiceImplTest.class.getClassLoader().getName()).toAbsolutePath();
+        var fileServiceImp = new FileServiceImpl(basePath);
+        Path testPath = Path.of("/it/is/here");
+        assertThrows(IllegalArgumentException.class, () -> fileServiceImp.getSecurePath(testPath));
```

Comment:
A good habit would be something like
```suggestion
        assertThatThrownBy( () -> fileServiceImp.getSecurePath(testPath))
                    .isInstanceOf(IllegalArgumentException.class)
                    .hasMessage("XXX");
```
What you are testing thus comes first, then what you are expecting. There are many variations for the chained methods.